### PR TITLE
Updated pwa content

### PR DESCRIPTION
**Pull request by jm-trd-ms**
Updated content to align with media folder clean-up.

### CODE REVIEW

File: microsoft-edge/progressive-web-apps-edgehtml/get-started.md [hunk 1]
```diff
@@ -72,78 +74,78 @@ If this were an existing live site, you could quickly generate a web app manifes
     }
     ```  
     
-    If this were a real PWA, you'd obviously want to customize at least the *name*, *start_url*, *short_name*, and *description*, as well as the *icons* \(we'll get to those next...\).  
+    If this were a real PWA, you should customize at least the *name*, *start_url*, *short_name*, and *description*, as well as the *icons* \(icons are covered in the next step\).
```
Comment (Reezaali) [inline microsoft-edge/progressive-web-apps-edgehtml/get-started.md:77]:
```suggestion
    If this were a real PWA, you would customize *name*, *start_url*, *short_name*, *description*, and *icons* \(icons are covered in the next step\).  
```

File: microsoft-edge/progressive-web-apps-edgehtml/get-started.md [hunk 2]
```diff
@@ -72,78 +74,78 @@ If this were an existing live site, you could quickly generate a web app manifes
...
     ```  
     
-    If this were a real PWA, you'd obviously want to customize at least the *name*, *start_url*, *short_name*, and *description*, as well as the *icons* \(we'll get to those next...\).  
+    If this were a real PWA, you should customize at least the *name*, *start_url*, *short_name*, and *description*, as well as the *icons* \(icons are covered in the next step\).  
     
-    See the [Web App Manifest][MDNWebAppManifest] reference on *MDN web docs* to learn more about the different member values and their purpose.  
+    To learn more about the different member values and associated purposes, see the [Web App Manifest][MDNWebAppManifest] reference.  
     
-1.  Next, let's fill in the empty `icons` array with actual image paths.  For that, we'll use PWA Builder's *App Image Generator*.  
+1.  Next, fill in the empty `icons` array with actual image paths.  For that, use the PWA Builder App Image Generator.
```
Comment (Reezaali) [inline microsoft-edge/progressive-web-apps-edgehtml/get-started.md:81]:
```suggestion
1.  Next, fill in the empty `icons` array with actual image paths by using the PWA Builder App Image Generator.  
```

File: microsoft-edge/progressive-web-apps-edgehtml/get-started.md [hunk 3]
```diff
@@ -72,78 +74,78 @@ If this were an existing live site, you could quickly generate a web app manifes
...
     
-    See the [Web App Manifest][MDNWebAppManifest] reference on *MDN web docs* to learn more about the different member values and their purpose.  
+    To learn more about the different member values and associated purposes, see the [Web App Manifest][MDNWebAppManifest] reference.  
     
-1.  Next, let's fill in the empty `icons` array with actual image paths.  For that, we'll use PWA Builder's *App Image Generator*.  
+1.  Next, fill in the empty `icons` array with actual image paths.  For that, use the PWA Builder App Image Generator.  
     
     1.  Using a web browser, download this [sample 512x512 PWA image][ImagePwa].  
-    1.  Go to the PWA Builder [App Image Generator][PwaBuilderAppImageGenerator], and select the *pwa.png* image you just saved as the **Input Image** and then click the **Download** button.  
+    1.  Go to the PWA Builder [App Image Generator][PwaBuilderAppImageGenerator], and select the `pwa.png` image you just saved as the **Input Image** and then click the **Download** button.
```
Comment (Reezaali) [inline microsoft-edge/progressive-web-apps-edgehtml/get-started.md:84]:
```suggestion
    1.  Go to the PWA Builder [App Image Generator][PwaBuilderAppImageGenerator], and select the `pwa.png` image you just saved as the **Input Image** and then choose the **Download** button.  
```

File: microsoft-edge/progressive-web-apps-edgehtml/get-started.md [hunk 4]
```diff
@@ -72,78 +74,78 @@ If this were an existing live site, you could quickly generate a web app manifes
...
 
 ![Web app manifest loading from localhost][ImageVsNodejsExpressManifest]  
 
-And one of your icons:  
+And one of your icons.  
 
 ![Square71x71Logo app logo loading from localhost][ImageVsNodejsExpressIcon]  
 
-If you publish the app live \(with an actual `start_url`\), the Bing search engine could now identify it as a candidate for [automatic packaging and submission to the Microsoft Store][PwaEdgehtmlMicrosoftStore] as an installable Windows 10 app.  Just make sure that your manifest includes the [*Quality signals for Progressive Web Apps*][WindowsBlogsPwaEdge] that Bing will be scanning for:  
+If you publish the app live \(with an actual `start_url`\), the Bing search engine now identifies it as a candidate for [automatic packaging and submission to the Microsoft Store][PwaEdgehtmlMicrosoftStore] as an installable Windows 10 app.  Just make sure that your manifest.json file includes the [Quality signals for Progressive Web Apps][WindowsBlogsPwaEdge] that Bing is scanning for the following.
```
Comment (Reezaali) [inline microsoft-edge/progressive-web-apps-edgehtml/get-started.md:104]:
```suggestion
If you publish the app live \(with an actual `start_url`\), the Bing search engine now identifies it as a candidate for [automatic packaging and submission to the Microsoft Store][PwaEdgehtmlMicrosoftStore] as an installable Windows 10 app.  Ensure that your manifest.json file includes the [Quality signals for Progressive Web Apps][WindowsBlogsPwaEdge] that Bing scans for as follows: 
```

File: microsoft-edge/progressive-web-apps-edgehtml/get-started.md [hunk 5]
```diff
@@ -72,78 +74,78 @@ If this were an existing live site, you could quickly generate a web app manifes
...
+If you publish the app live \(with an actual `start_url`\), the Bing search engine now identifies it as a candidate for [automatic packaging and submission to the Microsoft Store][PwaEdgehtmlMicrosoftStore] as an installable Windows 10 app.  Just make sure that your manifest.json file includes the [Quality signals for Progressive Web Apps][WindowsBlogsPwaEdge] that Bing is scanning for the following.  
 
 *   `name`  
 *   `description`  
-*   At least one icon 512px square or larger \(to ensure an image source of sufficient resolution for auto-generating you app's splash screen, store listing, tile image, and so on\).  
+*   At least one icon 512px square or larger \(to ensure an image source of sufficient resolution for auto-generating the splash screen of your app, store listing, tile image, and so on\).  
 
-...  in addition to being [HTTPS](#https), using [service workers](#service-workers), complying with [Microsoft Store Policies][LegalWindowsAgrementsMicrosoftStorePolicies].  
+In addition, it should be [HTTPS](#https), use [service workers](#service-workers), comply with [Microsoft Store Policies][LegalWindowsAgrementsMicrosoftStorePolicies].
```
Comment (Reezaali) [inline microsoft-edge/progressive-web-apps-edgehtml/get-started.md:110]:
```suggestion
In addition, use [HTTPS](#https), [service workers](#service-workers), and comply with [Microsoft Store Policies][LegalWindowsAgrementsMicrosoftStorePolicies].  
```

File: microsoft-edge/progressive-web-apps-edgehtml/get-started.md [hunk 6]
```diff
@@ -72,78 +74,78 @@ If this were an existing live site, you could quickly generate a web app manifes
...
 
-If you were to [publish this web app as a live site][VisualStudioNodejsTutorialPublishAzureAppService] \(for example, by setting up an [*Azure free account*][AzureCreateFreeAccount]\), you'll want to ensure your server is configured for HTTPS.  If you're using the [Microsoft Azure App Service][AzureWebApps] to host your site, it will be served over HTTPS by default.  
+If you [publish this web app as a live site][VisualStudioNodejsTutorialPublishAzureAppService] \(for example, by setting up an [Azure free account][AzureCreateFreeAccount]\), you must ensure your server is configured for HTTPS.  If you use the [Microsoft Azure App Service][AzureWebApps] to host your site, it is served over HTTPS by default.  
 
-For this guide we'll continue using *http://localhost* as a placeholder for a live site served over *https://*.  
+For this guide, you continue using `http://localhost` as a placeholder for a live site served over `https://`.
```
Comment (Reezaali) [inline microsoft-edge/progressive-web-apps-edgehtml/get-started.md:118]:
```suggestion
For this guide, continue using `http://localhost` as a placeholder for a live site served over `https://`.  
```

File: microsoft-edge/progressive-web-apps-edgehtml/get-started.md [hunk 7]
```diff
@@ -72,78 +74,78 @@ If this were an existing live site, you could quickly generate a web app manifes
...
+For this guide, you continue using `http://localhost` as a placeholder for a live site served over `https://`.  
 
 ### Service Workers  
 
-*Service Workers* is the key technology behind PWAs.  They act as a proxy between your PWA and the network, enabling your website to act as an installed native app: serving up offline scenarios, responding to server push notifications, and running background tasks.  Service workers also open up all kinds of new performance strategies; and website need not even be full-blown web app to take advantage of the service worker cache for fine-tuned page load performance.  
+Service Workers is the key technology behind PWAs and act as a proxy between your PWA and the network, enabling your website to act as an installed native app including serving up offline scenarios, responding to server push notifications, and running background tasks.  Service workers also open up all kinds of new performance strategies.  You are not required to implement a full-blown web app to take advantage of the service worker cache for fine-tuned page load performance for your website.
```
Comment (Reezaali) [inline microsoft-edge/progressive-web-apps-edgehtml/get-started.md:122]:
```suggestion
Service Workers is the key technology behind PWAs. They act as a proxy between your PWA and the network to enable your website to function as an installed native app that can serve up offline scenarios, respond to server push notifications, and run background tasks.  Service workers also open up new performance strategies.  You are not required to implement a full web app to use the service worker cache for fine-tuned page load performance for your website.  
```

File: microsoft-edge/progressive-web-apps-edgehtml/get-started.md [hunk 8]
```diff
@@ -72,78 +74,78 @@ If this were an existing live site, you could quickly generate a web app manifes
...
 
-For this tutorial, we'll use a ready-made "Offline page" service worker script courtesy of [PWA Builder][PwaBuilderServiceWorker].  From this, you can go on to customize it with more elaborate functionality according to your needs for performance, network bandwidth, and so on.  Check out Mozilla's [Service Worker Cookbook][ServiceWorkerCookbook] for a number of useful service worker caching "recipe" ideas.  
+For this tutorial, use a ready-made Offline page service worker script courtesy of [PWA Builder][PwaBuilderServiceWorker].  From this, you should customize it with more elaborate functionality according to your needs for performance, network bandwidth, and so on.  Check out the [Service Worker Cookbook][ServiceWorkerCookbook]  provided by Mozilla for a number of useful service worker caching-recipe ideas.
```
Comment (Reezaali) [inline microsoft-edge/progressive-web-apps-edgehtml/get-started.md:128]:
```suggestion
For this tutorial, use the Offline page service worker script on [PWA Builder][PwaBuilderServiceWorker].  Start by customizing the script with more functionality according to your performance requirements, network bandwidth, and so on.  Check out the [Service Worker Cookbook][ServiceWorkerCookbook]  provided by Mozilla for a number of useful service worker caching ideas.  
```

File: microsoft-edge/progressive-web-apps-edgehtml/get-started.md [hunk 9]
```diff
@@ -72,78 +74,78 @@ If this were an existing live site, you could quickly generate a web app manifes
...
+1.  Open the download folder and copy the following two files  
 
     *   ServiceWorker1\pwabuilder-sw-register.js  
     *   ServiceWorker1\pwabuilder-sw.js  
     
-    ...  to the *public* folder of your Visual Studio web app project.  \(From Visual Studio, use Ctrl+O to open file explorer to your project and navigate to the *public* folder\).  
+    Save the files to the `public` folder of your Visual Studio web app project.  \(From Visual Studio, use `Ctrl`+`O` to open file explorer to your project and navigate to the `public` folder\).  
     
-    Its worth reviewing the code in both of these files, to get the gist of how to register a service worker that caches a designated page \(*offline.html*\) and serves it when a network fetch fails.  Next, we need to create a simple "offline.html" page as a placeholder for our app's offline functionality.  
+    it is worth reviewing the code in both of these files, to get the gist of how to register a service worker that caches a designated page \(`offline.html`\) and serves it when a network fetch fails.  Next, create a simple `offline.html` page as a placeholder for the offline functionality of your app.
```
Comment (Reezaali) [inline microsoft-edge/progressive-web-apps-edgehtml/get-started.md:138]:
```suggestion
    It is worth reviewing the code in both of these files, to get the gist of how to register a service worker that caches a designated page \(`offline.html`\) and serves it when a network fetch fails.  Next, create a simple `offline.html` page as a placeholder for the offline functionality of your app.  
```

File: microsoft-edge/progressive-web-apps-edgehtml/get-started.md [hunk 10]
```diff
@@ -159,153 +161,170 @@ For this tutorial, we'll use a ready-made "Offline page" service worker script c
...
     
-    1.  Open the Edge DevTools **Console** \(`Ctrl`+`Shift`+`J`\) and verify the *Service worker was registered*.  
-    1.  In the **Debugger** panel, expand the **Service Workers** control and click on your origin.  In the *Service Worker Overview*, verify your service worker is activated and running:  
+    1.  Open the Edge DevTools **Console** \(`Ctrl`+`Shift`+`J`\) and verify the Service worker was registered.  
+    1.  In the **Debugger** panel, expand the **Service Workers** control and click on your origin.  In the **Service Worker Overview**, verify your service worker is activated and running.  
         
         ![Edge DevTools Service Worker overview][ImageDevtoolsSwOverview]  
         
-    1.  Still in the Debugger, expand the **Cache** control and verify that the *offline.html* page has been cached.  
+    1.  Still in the Debugger, expand the **Cache** control and verify that the `offline.html` page has been cached.
```
Comment (Reezaali) [inline microsoft-edge/progressive-web-apps-edgehtml/get-started.md:185]:
```suggestion
    1.  Expand the **Cache** control and verify that the `offline.html` page is cached.  
```

File: microsoft-edge/progressive-web-apps-edgehtml/get-started.md [hunk 11]
```diff
@@ -314,124 +333,106 @@ The following is adapted from the *Push Rich Demo* in Mozilla's [Service Worker
...
     
-    Check out the MDN documentation on [`PushManager`][MDNPushManager] and NPM docs on [`Web-Push`][NPMWebPushUsage] for more details on how these APIs work and their various options.  
+    As your service worker activates and attempts to subscribe your PWA to push notifications, you should see a permission dialog at the bottom of the page.  
     
-1.  **Set up push and notificationclick event handlers.**  
+    ![Permission dialog for enabling notifications][ImageNotificationPermission]  
     
-    With our push subscription set up, the remainder of the work happens in the service worker.  First we need to set up a handler for server-sent push events, and respond with a toast notification \(if permission was granted\) displaying the push data payload.  Next we'll add a click handler for the toast to dismiss the notification and sort through a list of currently open windows to open and/or focus the intended PWA client page.  
+    Click **Yes** to enable toast notifications for your PWA.
```
Comment (Reezaali) [inline microsoft-edge/progressive-web-apps-edgehtml/get-started.md:408]:
```suggestion
    Choose **Yes** to enable toast notifications for your PWA.  
```

File: microsoft-edge/progressive-web-apps-edgehtml/get-started.md [hunk 12]
```diff
@@ -314,124 +333,106 @@ The following is adapted from the *Push Rich Demo* in Mozilla's [Service Worker
...
+    As your service worker activates and attempts to subscribe your PWA to push notifications, you should see a permission dialog at the bottom of the page.  
     
-1.  **Set up push and notificationclick event handlers.**  
+    ![Permission dialog for enabling notifications][ImageNotificationPermission]  
     
-    With our push subscription set up, the remainder of the work happens in the service worker.  First we need to set up a handler for server-sent push events, and respond with a toast notification \(if permission was granted\) displaying the push data payload.  Next we'll add a click handler for the toast to dismiss the notification and sort through a list of currently open windows to open and/or focus the intended PWA client page.  
+    Click **Yes** to enable toast notifications for your PWA.  
     
-    In your *pwabuilder-sw.js* file, append the following handlers:  
+1.  From the Service Worker Overview pane, try clicking the  **Push** button.  A toast notification with the \(hard-coded "Test push message from DevTools"\) payload should appear.
```
Comment (Reezaali) [inline microsoft-edge/progressive-web-apps-edgehtml/get-started.md:410]:
```suggestion
1.  From the Service Worker Overview pane, try choosing the  **Push** button.  A toast notification with the \(hard-coded "Test push message from DevTools"\) payload should appear.  
```

File: microsoft-edge/progressive-web-apps-edgehtml/get-started.md [hunk 13]
```diff
@@ -314,124 +333,106 @@ The following is adapted from the *Push Rich Demo* in Mozilla's [Service Worker
...
-                var client = clientList[i];
-                if (client.url == 'http://localhost:1337/' && 'focus' in client)
-                    return client.focus();
-            }
-            if (clients.openWindow)
-                return clients.openWindow('/');
-        }));
-    });
-    ```  
+1.  Next try clicking the **Send Notification** button on the homepage of your PWA.  This time a toast with the payload from your server appears.
```
Comment (Reezaali) [inline microsoft-edge/progressive-web-apps-edgehtml/get-started.md:414]:
```suggestion
1.  Next try choosing the **Send Notification** button on the homepage of your PWA.  This time a toast with the payload from your server appears.  
```

File: microsoft-edge/progressive-web-apps-edgehtml/get-started.md [hunk 14]
```diff
@@ -314,124 +333,106 @@ The following is adapted from the *Push Rich Demo* in Mozilla's [Service Worker
...
+*   [PWAs in the Microsoft Store][PwaEdgehtmlMicrosoftStore].  Learn more about the benefits of app store distribution and how to submit your PWA.  
 
-Also be sure to explore the great resources from around the dev community!  [MDN web docs][MDNProgressiveWebApps] and [Google Developers][WebDevProgressiveWebApps] have excellent guides on Progressive Web Apps.  [https://pwa.rocks][ProgressiveWebApps] showcases real-world examples of PWAs, and [https://hnpwa.com][HackerNewsProgressiveWebApps] compares different frameworks and performance patterns for implementing a sample \(*Hacker News* reader\) PWA.  
+Also be sure to explore the great resources from around the dev community!  [MDN web docs][MDNProgressiveWebApps] and [Google Developers][WebDevProgressiveWebApps] have excellent guides on Progressive Web Apps.  [https://pwa.rocks][ProgressiveWebApps] showcases real-world examples of PWAs, and [https://hnpwa.com][HackerNewsProgressiveWebApps] compares different frameworks and performance patterns for implementing a sample \(Hacker News reader\) PWA.  
 
 <!-- image links -->
```
Comment (Reezaali) [inline microsoft-edge/progressive-web-apps-edgehtml/get-started.md:438]: Can we include a See Also section with related links?

Comment (jm-trd-ms) [reply microsoft-edge/progressive-web-apps-edgehtml/get-started.md:438]: It is possible. 
I will start a thread.

Comment (jm-trd-ms) [reply microsoft-edge/progressive-web-apps-edgehtml/get-started.md:443]: See also implemented.

**Comment by jm-trd-ms** [comment]
$sign-off and merged.

**Pull request merged**